### ISSUE TITLE
Update Table 2 of QA Feedback Reports for No Errors Submissions

### ISSUE THREAD

**Issue by lgiannini1** [US-EPA-CAMD/PI-1#6422]
The QA Submission Feedback Reports for submissions that include QA files that evaluated with no errors do not list the correct test/ QCE/ TEE information in Table 2. 

**Acceptance Criteria:** All tests submitted should be listed, regardless of their evaluation status.

AC1. ORIS 7900, SH4 Linearity test LINE-Q32023-422-8 was submitted with no errors (Submission ID b72a9e3b-1648-40b0-bb6d-3a07316edcb1). This test does not appear in Table 2.

![Image](https://github.com/user-attachments/assets/2dfdb238-f004-440e-9529-0ef91b86e7e6)

AC2. ORIS 7900, SH3 Linearity LINE-Q32023-013-5 was submitted with informational messages and RATA RATA-Q32023-NOX-1 was submitted with admin overrides (Submission ID c6e94aeb-5e2d-4f07-a8ce-fe6ed713af63). Both tests appear in Table 2.

![Image](https://github.com/user-attachments/assets/376633ad-08c9-4431-91ca-e5cfa4cc7235)

AC3. ORIS 7900, SH4 RATA RATA-Q32023-NX4-1 was submitted with admin overrides and Linearity LINE-Q32023-412-7 was submitted with no errors (ca6fde82-13e9-452c-81ce-0267746ffa0c). The linearity was does not appear in Table 2.

![Image](https://github.com/user-attachments/assets/17d13e0a-c64e-4f8b-b140-30370550cd9f)

**Comment by EvanSun220** [comment]
### **Issue resolved.**
Here are the screenshots of the feedback report email I received now:
For **ORIS 7900, SH4 Linearity test LINE-Q32023-422-8**, the test appeared on the Table 2, means test passed.
![Image](https://github.com/user-attachments/assets/e7760080-1aaf-451f-b074-58a28491b784)

For **ORIS 7900, SH4 RATA RATA-Q32023-NX4-1** and **Linearity LINE-Q32023-412-7**, both of the tests appeared on the Table 2, means test passed.
![Image](https://github.com/user-attachments/assets/9799e50e-fa71-4a0d-acfd-056b8aeb33f4)

**Comment by lgiannini1** [comment]
Currently unable to import QA test data in the tst environment because of the following error. Will create a new ticket for this issue.

![Image](https://github.com/user-attachments/assets/c2975155-1b89-4dcc-b624-dee2e3d5b9f2)

**Comment by annalbrecht** [comment]
Error potentially fixed in #6517.

**Comment by ibarra-michelle** [comment]
@mxtomoto1 , just checking before I start any testing - are you working on testing this ticket? It has a EPA QA label but perhaps you picked it up to close out.

**Comment by ibarra-michelle** [comment]
> Error potentially fixed in [#6517](https://github.com/US-EPA-CAMD/easey-ui/issues/6517).

Unable to test at the moment since #6517 is currently under peer review (Sprint 15 tix). The blocked label has been added and will be removed after #6517 has been pushed into the test env.  Currently, this is the error I receive:

<img width="409" alt="Image" src="https://github.com/user-attachments/assets/0646fb9a-02ad-4e6f-9282-82523e8b8b0f" />

FYA @annalbrecht @EvanSun220 @alangmaid @spetros-do

**Comment by mark-hayward-erg** [comment]
@ibarra-michelle the update for #6517 has been deployed in the test environment.  I have removed the "blocked" label from this ticket.

**Comment by ibarra-michelle** [comment]
@EvanSun220 @lgiannini1 @mark-hayward-erg @esaber76 

Unable to verify in test for two different facility+configs.

**Summary:** For the three different Acceptance Criteria, I received evaluation errors. I logged in with my admin account, then tried submitting Error Submission Overrides. I was able to successfully submit overrides for a MP errors and could not submit overrides for the QA errors. From what I can see, the Test Type drop down does not populate and a Forbidden Resource error appeared. See the two screenshot below with the Forbidden Resource error for ORIS 7900 and ORIS 2480.

At the DSU, the team discussed and agreed to move this ticket #6422 to QA Done, then I create a new ticket (#6579) to address the ECMPS Admin bug, which will includes retest the acceptance criteria in this ticket. 

## ORIS 7900 Config SH4
<img width="420" alt="Image" src="https://github.com/user-attachments/assets/51a57c8e-5491-4ccf-87eb-1f662ba5b4c0" />

## ORIS 2480 Config 4
<img width="824" alt="Image" src="https://github.com/user-attachments/assets/c5483299-d39a-4e53-8ea0-1e6664823292" />

**Comment by esaber76** [comment]
@ibarra-michelle - the RATA test I provided for ORIS 2480 should automatically return an ADMNOVR for the evaluation. You don't need to add a suppression record.

**Comment by ibarra-michelle** [comment]
> [@ibarra-michelle](https://github.com/ibarra-michelle) - the RATA test I provided for ORIS 2480 should automatically return an ADMNOVR for the evaluation. You don't need to add a suppression record.

@esaber76 thx for confirming I shouldn't need a suppression record. However, I am still unable to select the record in the Submission module without any Error Suppressions added. Below are screenshots of the Eval Report, Eval module displaying the QA test's Eval status (Information Message), and Submission Module. 

## Evaluation Report
<img width="910" alt="Image" src="https://github.com/user-attachments/assets/849bf82d-be3f-49c7-b230-ee84d8207abe" />

## Evaluation Module listing ORIS 2480 Config 4
<img width="801" alt="Image" src="https://github.com/user-attachments/assets/45ce788b-eda7-4b8f-98d9-d6940351c19d" />

## Submission Module  listing ORIS 2480 Config 4
<img width="763" alt="Image" src="https://github.com/user-attachments/assets/cef9a663-2036-436b-b05e-6102e5d4fab7" />

**Comment by esaber76** [comment]
@ibarra-michelle - you would need to first use the Sys Admin tool to reopen the submission window for that test (and any other tests you may be submitting along with it).  

Also, if you need to generate an informational error on a linearity test as described in the ticket description, reopen the window for that test and change the test result code to aborted.

**Comment by ibarra-michelle** [comment]
Verified in Test.

Thanks @esaber76 ! I was able to submit for ORIS 2480 Config 4, obtain a Feedback report and see table 2 includes the submitted tests. This helped me test and verify the original three Acceptance Criteria associated with this ticket.  

AC1. ORIS 7900, SH4 Linearity test LINE-Q32023-422-8 was submitted successfully. The single test appears in table 2:
<img width="941" alt="Image" src="https://github.com/user-attachments/assets/6794edf2-2b98-43cd-9e21-4a0992840c1b" />

AC2. Verified in test: ORIS 7900, SH3 Linearity LINE-Q32023-013-5 and RATA RATA-Q32023-NOX-1was submitted successfully. Both tests appear in Table 2.
<img width="635" alt="Image" src="https://github.com/user-attachments/assets/33031c9f-66e8-4dba-b061-a9d11f47d889" />

AC3. ORIS 7900, SH4 RATA RATA-Q32023-NX4-1 and Linearity LINE-Q32023-412-7 was submitted successfully. Both tests appear in Table 2: 
<img width="635" alt="Image" src="https://github.com/user-attachments/assets/c46c2fa2-fa4e-4148-b928-76e830da1de3" />

FYA @lgiannini1 @mark-hayward-erg @EvanSun220
